src/ts/docs/help/onboarding.ts

```diff
@@ -369,7 +369,7 @@ export default {
 					description: translate('onboardingCollaborationText'),
 					noButton: true,
 					param: {
-						element: '#category-collaboration',
+						element: '#popup-usecase #category-collaboration',
```

Comment:
@ra3orblade this stopped working with this selector. I haven't found the reason yet.

Reply:
Sorry, I thought you would check the id as I do not remember the correct string, it's being camelCased so the correct one would be `#popupUsecase`.

Reply:
This explains it! :D